composer.json

```diff
@@ -19,10 +19,10 @@
         "doctrine/orm": "^2.5.3",
         "doctrine/annotations": "^1.2",
         "doctrine/phpcr-bundle": "^1.3.5",
-        "doctrine/doctrine-bundle": "~1.0",
+        "doctrine/doctrine-bundle": "~1.6",
```

Comment:
Necessary in order to fix the doctrine listeners. With lower versions it is not possible to use `$entityManager->getRepository(TargetGroupRuleInterface::class)`, because `ResolveTargetEntitiesListener` is not working.